File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.*;
import java.util.Scanner;

public class Main{
    static int[] arr = new int[9];
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int sum =0;
        for (int i=0; i<9; i++){
            arr[i] = sc.nextInt();
            sum += arr[i];
        }
        int[] flag = new int[2];
        for (int i=0; i<9; i++){
            for (int j= i+1; j<9; j++){
                if (sum - (arr[i] + arr[j]) == 100){
                    flag[0] = i; flag[1] = j;
                }
            }
        }
        List<Integer> ans = new ArrayList<>();
        for (int i=0; i<9; i++){
            if (i == flag[0] || i == flag[1])continue;
            ans.add(arr[i]);
        }
        Collections.sort(ans);
        for (int num : ans){
            System.out.println(num);
        }
    }
}

File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.kt
import java.util.Scanner

var arr : Array<Int> = Array(9){0};
var psum :Int =0;
fun main(){
    val sc = Scanner(System.`in`)
    for (i in 0..<9){
        var temp : Int = sc.nextInt();
        psum += temp;
        arr[i] = temp;
    }
    var a : Int = 0;
    var b: Int =0;
    for (i in 0..<9){
        for (j in i+1..<9){
            if (psum-(arr[i] + arr[j]) == 100){
                a = i;
                b = j;
            }
        }
    }
    var ret : Array<Int> = Array(7){0};
    var j : Int =0;
    for (i in 0..<9){
        if (i == a || i == b)continue;
        ret[j] = arr[i];
        j++;
    }
    ret.sort();
    for (i in ret){
        println(i);
    }
}